Add documentation for new WMS WebControl pro integration (#34353)
home-assistant/home-assistant.github.io: source/_integrations/wmspro.markdown

---
title: WMS WebControl pro
description: Instructions on how to integrate WAREMA devices via WMS WebControl pro within Home Assistant.
ha_category:
  - Cover
  - Hub

> ha_release: '2024.9'

Reviewer: ```suggestion
ha_release: "2024.10"
```

ha_iot_class: Local Polling
ha_codeowners:
  - '@mback2k'
ha_domain: wmspro
ha_config_flow: true
ha_platforms:
  - cover
ha_integration_type: integration
related:
  - url: https://www.warema.com/en/smart-home/wms-webcontrol-pro/
    title: Consumer information about WMS WebControl pro
  - url: https://smartbuildings.warema.com/en/control-systems/radio-systems/wms/wms-webcontrol-pro/
    title: Technical documentation for WMS WebControl pro
---

> The WMS WebControl pro integration allows users to integrate their WAREMA devices into Home Assistant.

Reviewer 2: ```suggestion
The **WMS WebControl pro** {% term integration %} allows you to integrate WAREMA devices into Home Assistant.
```
